server.js

```diff
@@ -10,38 +1,47 @@
+const express = require("express");
+var app = express();
+var server = require("http").Server(app);
+var io = require("socket.io")(server);
+const port = 6969;
+
+app.get("/", (req, res) => {
+  res.sendFile("templates/client_chat.html", { root: __dirname });
+});
 
-app.get('/darkmode.css', (req, res) => {
-    res.sendFile('darkmode.css', { root: __dirname });
-})
+app.get("/darkmode.css", (req, res) => {
+  res.sendFile("darkmode.css", { root: __dirname });
+});
 
-app.get('/manifest.webmanifest', (req, res) => {
-    res.sendFile('manifest.webmanifest', { root: __dirname });
-})
+app.get("/manifest.webmanifest", (req, res) => {
+  res.sendFile("manifest.webmanifest", { root: __dirname });
+});
 
-app.get('/sw.js', (req, res) => {
-    res.sendFile('sw.js', { root: __dirname });
-})
+app.get("/sw.js", (req, res) => {
+  res.sendFile("sw.js", { root: __dirname });
+});
 
-app.get('/icons/512', (req, res) => {
-    res.sendFile('icons/cryptochat 512.png', { root: __dirname });
-})
+app.get("/icons/512", (req, res) => {
+  res.sendFile("icons/cryptochat 512.png", { root: __dirname });
+});
 
-app.get('/icons/192', (req, res) => {
-    res.sendFile('icons/cryptochat 192.png', { root: __dirname });
-})
+app.get("/icons/192", (req, res) => {
+  res.sendFile("icons/cryptochat 192.png", { root: __dirname });
+});
 
-app.get('/notification', (req, res) => {
-    res.sendFile('notification.mp3', { root: __dirname });
-})
+app.get("/notification", (req, res) => {
+  res.sendFile("notification.mp3", { root: __dirname });
+});
 
-app.get('/favicon.ico', (req, res) => {
-    res.sendFile('icons/favicon.ico', { root: __dirname });
-})
+app.get("/favicon.ico", (req, res) => {
+  res.sendFile("icons/favicon.ico", { root: __dirname });
+});
 
-io.on('connection', connection =>  {
-    connection.on('chat event', data => {
-        connection.emit('my response', data)
-    });
+io.on("connection", (connection) => {
+  connection.on("chat event", (data) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/CryptoChat-dev/cryptochat/pullRequest?prid=7237627)
```suggestion
    connection.emit('my response', data);
```